burstphoto/align.metal

```diff
@@ -156,12 +326,15 @@ kernel void compute_tile_differences(texture2d<float, access::read> ref_texture
             // compute the indices of the pixels to compare
             int ref_tile_x = x0 + dx1;
             int ref_tile_y = y0 + dy1;
-            int comp_tile_x = x0 + dx0 + dx1;
-            int comp_tile_y = y0 + dy0 + dy1;
+            int comp_tile_x = ref_tile_x + dx0;
+            int comp_tile_y = ref_tile_y + dy0;
+            
             // if the comparison pixels are outside of the frame, attach a high loss to them
             if ((comp_tile_x < 0) || (comp_tile_y < 0) || (comp_tile_x >= texture_width) || (comp_tile_y >= texture_height)) {
-                diff += 2;
+                
+                diff += abs(ref_texture.read(uint2(ref_tile_x, ref_tile_y)).r + 2*UINT16_MAX_VAL);
```

Comment:
Is there any point to `ref_texture.read(uint2(ref_tile_x, ref_tile_y)).r` if you're adding `2*UINT16_MAX_VAL`? Can't this be replaced by only `2*UINT16_MAX_VAL` or something huge?

Reply:
Yes, you are correct, it could be replaced with a huge constant value. However, for consistency reasons with the optimized version of this function compute_tile_differences25(), where I could not find a way to only add a constant, I left it as it is.

Reply:
Sorry for being not precise. I mean consistency of the produced value.